lego/apps/events/views.py

```diff
@@ -48,7 +48,9 @@ def get_queryset(self):
                 'pools', 'pools__registrations', 'company', 'tags'
             )
         elif self.action == 'retrieve':
-            queryset = Event.objects.prefetch_related('pools', 'pools__permission_groups', 'tags')
+            queryset = Event.objects.prefetch_related(
+                'pools', 'pools__permission_groups', 'tags', 'company', 'responsible_group'
```

Comment:
shouldn't `responsible_group` be in a `select_related` since it's on the event model? and not a many to many field

Reply:
Yeah my bad, to fast